service/controller/cluster.go

```diff
@@ -84,8 +82,8 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 			Logger:  config.Logger,
 			Watcher: config.G8sClient.ProviderV1alpha1().AzureConfigs(""),
 
-			//ResyncPeriod: informer.DefaultResyncPeriod,
-			ResyncPeriod: 1 * time.Minute,
+			RateWait:     informer.DefaultRateWait,
+			ResyncPeriod: informer.DefaultResyncPeriod,
```

Comment:
Do we even need to set those?

Reply:
We do not need to do it but I would like to do it to be explicit and have it easier to change them for testing.